helix-term/src/keymap.rs

```diff
@@ -520,9 +520,10 @@ impl Default for Keymaps {
                 "r" => goto_reference,
                 "i" => goto_implementation,
                 "t" => goto_window_top,
-                "m" => goto_window_middle,
+                "c" => goto_window_middle,
```

Comment:
I rejected changing this keybind before :P why not find a different keymap for `goto_last_modified_file`?

Reply:
There are a couple of reasons

- it is the same as kakoune now `gc`
- it is more aligned with view which is `zc` which have the same behavior

Reply:
Not sure if we'd want to put this in the default keymap but I'd probably remap `goto_last_modified_file` to `gM`.